packages/core/demo/demo-data/pie-donut.ts

```diff
@@ -4,7 +4,9 @@ export const pieOptions = {
 	accessibility: false,
 	legendClickable: true,
 	containerResizable: true,
-	colors
+	colors,
+	title: "Pie Chart",
+	theme: "dark"
```

Comment:
How many themes are we required to support?
If there are multiple, themes should be exported in the library.

e.g.
```js
import { themes } from "@carbon/charts";

const chartOptions = {
  theme: themes.G90,
  theme: themes.G30,
  theme: themes.G20,
  theme: themes.P90,
  theme: themes.DARK
}
```

Similar to that^^

Reply:
im going tol remove the theme in favour of it being in its own PR

Reply:
👍

Reply:
Let's remove this line then in favour of the PR you mentioned